src/app/features/genes/components/gene-nominations/gene-nominations.component.ts

```diff
@@ -36,28 +36,33 @@ export class GeneNominationsComponent {
     }
 
     this.teamService.getTeams().subscribe((res: TeamsResponse) => {
-      const nominations: NominatedTarget[] = [];
       const teams: Team[] = res.items;
-      const teamNames = this._gene?.nominatedtarget?.map((n) => n.team);
```

Comment:
Bug must've had to do with an assumption that multiple nominations were not needed so the sorting of nominations by teams was changed to matching teams on the nomination list.